packages/react-reconciler/src/ReactFiberOffscreenComponent.js

```diff
@@ -19,6 +19,7 @@ export type OffscreenProps = {|
   // called "Offscreen." Possible alt: <Visibility />?
   mode?: 'hidden' | 'visible' | null | void,
   children?: ReactNodeList,
+  cleanupPassiveEffects?: boolean,
```

Comment:
Was thinking about incorporating this into `mode`, but I couldn't think of a good name. Open to suggestions!

Reply:
Yeah let's make a mode. Name isn't that important for now, so I'd err on the side of the obnoxiously specific, like `hidden-with-aggressive-cleanup`.